deepinv/utils/demo.py

```diff
@@ -78,9 +79,27 @@ def get_image_url(file_name):
     )
 
 
+def get_data_home():
+    """Return folder to store data in.
+
+    This folder can be set by setting
```

Comment:
```suggestion
    This folder can be set by setting the environment variable``DEEPINV_DATA``,
    or ``XDG_DATA_HOME``. By default, it will be set in ``./datasets``.
```

Reply:
I think there has been some issue in accepting comments as they were deleted in the new version of the docs?